scripts/prepare-release.sh

```diff
@@ -33,7 +33,11 @@ for arch in `ls -1 $BIN_DIR/`;do
 
     # Create a tar.gz of the binary
     echo "gzipping binary $source_file as $target_file"
-    tar -czvf $target_file.tar.gz --directory=$source_dir $source_filename
+    if ["$suffix" == "exe"]; then
+        tar -czvf $target_file.zip --directory=$source_dir $source_filename
```

Comment:
I don't think that this will work. 
For creating a zip archive you will need to use `zip` command

Reply:
I was able to tar it with zip extension on a linux box using that tar command, then I was able to unzip it on a windows 10 box using 7zip, are you OK with that?